sys/arm64/include/pte.h

```diff
@@ -101,11 +102,10 @@ typedef	uint64_t	pt_entry_t;		/* page table entry */
 
 #if __has_feature(capabilities)
 /*
- * The aarch64 orr instruction cannot handle ATTR_DEFAULT |
- * ATTR_CAP_RW as a single operand, so separate orr instructions are
- * required for ATTR_CAP_RW.
+ * The aarch64 orr instruction cannot handle ATTR_CAP_RW as
+ * a single operand, so separate orr instructions are required.
  */
-#define	ATTR_CAP_RW	(ATTR_LC_ENABLED | ATTR_SC)
+#define	ATTR_CAP_RW	(ATTR_LC_ENABLED | ATTR_CDBM)
```

Comment:
Ah, hmm.  Is `ATTR_CAP_RW` used anywhere still after this?  (I guess we might use it in C?)

Reply:
Yes, it's used in `pmap.c`.  It's always guarded by `#ifdef __has_feature(capabilities)` and I wonder if it'd be better to guard the definition and make it 0 on non-caps?  But that's minor and can be done later, if it seems like a good plan.

Reply:
I think I prefer keeping it undefined for !capabilities and explicit #ifdef's for its use as is currently done.

Reply:
0 for !capabilities doesn't help you in assembly, since `0` (and `-1`) are not immediates that can be encoded for `orr`, I guess for obvious reasons.